### ISSUE TITLE
Right-click Context Menu Appears broken when this module is installed.

### ISSUE THREAD

**Issue by thorgibrewer** [WBHarry/pf2e-bestiary-tracking#125]
I cannot right-click a chat card and get the context window to appear. usually used to reroll using hero point but no context window pops up. I turn off the module and it works. Console screenshot below when I right-click a chat card.

![image](https://github.com/user-attachments/assets/0b1fa4fb-72d8-403f-9e18-42c6fa8b7307)

**Comment by WBHarry** [comment]
The module makes use of a hidden Journal to store data. That Journal is appearantly missing in your world, so nothing should work atm, really.

You must either be updating from a quite old version (there was an issue with this at one point), or some other module or handling led to deleting the journal despite it being hidden.

-----
Regardless! Easy to get it going again. Just use the `Bestiary Reset` macro included with the module. It'll clear out all Bestiary data and set it up properly again. 🤖 
![image](https://github.com/user-attachments/assets/10d45a92-09b4-45a8-8fbf-ac68b48dfcf4)

**Comment by thorgibrewer** [comment]
thanks! that worked. I think that makes sense, my main game I run weekly didn't have the problem in it's world guess I've updated/migrated or reinitialized the bestiary tracker updates as they have come out, but for the other two worlds they've gone longer without play so it makes sense they got borked.

**Comment by thorgibrewer** [comment]
Consider this closed from my side!

**Comment by WBHarry** [comment]
Yeah, just a bad thing that happened at one point in development. Shouldn't be any issues like that since a good while back 🤖 🎉
